docs/local_subscriber.md

```diff
@@ -0,0 +1,24 @@
+LocalSubscriber
+===============
+The LocalSubscriber block subscribes to the configured topic and outputs signals received. Only LocalPublisher blocks on the same nio instance can send data to the LocalSubscriber blocks. Unlike the Subscriber block, these signals do not need to contain data that is valid JSON.
+
+Properties
+----------
+- **local_identifier**: Hidden property with a default of `[[INSTANCE_ID]]`. Unique identifier of this instance in the nio system.
```

Comment:
Hidden properties can be omitted, if it can't be seen in the designer it doesn't need to be here I think.